test/performance/specs/site-editor.spec.js

```diff
@@ -14,13 +14,14 @@ const path = require( 'path' );
 const {
 	readFile,
 	deleteFile,
-	saveResultsFile,
 	getTraceFilePath,
 	getTypingEventDurations,
 	getLoadingDurations,
 	loadBlocksFromHtml,
 } = require( '../utils' );
 
+const BROWSER_IDLE_WAIT = 1000;
```

Comment:
ℹ️ Based on https://github.com/WordPress/gutenberg/issues/51383#issuecomment-1613460429

Reply:
Let's move this to an inline comment?